src/spikeinterface/curation/auto_merge.py

```diff
@@ -356,6 +362,12 @@ def compute_merge_unit_groups(
             )
             outs["pairs_decreased_score"] = pairs_decreased_score
 
+        elif step == "slay_score":
+
+            M_ij = compute_slay_matrix(sorting_analyzer, params["k1"], params["k2"], params["template_diff_thresh"])
```

Comment:
I think we can use the `pair_mask` from template similarity as input, to make use of the multiple steps (and give the option to the user to add more!):
```suggestion
            M_ij = compute_slay_matrix(sorting_analyzer, params["k1"], params["k2"], pair_mask=pair_mask)
```